find_new/src/find_new.py

```diff
@@ -83,13 +83,12 @@ def submit_jobs_for_granule(hyp3, event_id, granule):
 
     try:
         neighbors = asf_search.get_nearest_neighbors(granule['granuleName'])
+    except ASFSearchError:
+        raise GranuleError()
     except requests.HTTPError as e:
```

Comment:
This i weird

Reply:
Agreed.  https://github.com/ASFHyP3/hyp3-sdk/issues/97 is open on the hyp3-sdk side to track it.